showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITDynamicRoutingHeaders.java

```diff
@@ -114,16 +121,22 @@ public void onClose(int statusCode, HttpJsonMetadata trailers) {
 
   private HttpJsonCapturingClientInterceptor httpJsonInterceptor;
   private GrpcCapturingClientInterceptor grpcInterceptor;
-
+  private GrpcCapturingClientInterceptor grpcComplianceInterceptor;
   private EchoClient grpcClient;
   private EchoClient httpJsonClient;
+  private ComplianceClient grpcComplianceClient;
+
 
   @Before
   public void createClients() throws Exception {
     // Create gRPC Interceptor and Client
     grpcInterceptor = new GrpcCapturingClientInterceptor();
     grpcClient = TestClientInitializer.createGrpcEchoClient(ImmutableList.of(grpcInterceptor));
 
+    // Create gRPC ComplianceClient and Interceptor
+    grpcComplianceInterceptor = new GrpcCapturingClientInterceptor();
+    grpcComplianceClient = TestClientInitializer.createGrpcComplianceClient(ImmutableList.of(grpcComplianceInterceptor));
```

Comment:
nit: Add a small comment above that we include compliance client in this IT because the EchoClient doesn't have an implicit enum routing test case. Possibly link the ticket in gapic-showcase to here.

Reply:
Done.